app/templates/page-label.html

```diff
@@ -0,0 +1,23 @@
+<gu-row>
+  <gu-column span="6" gu-box="primary">
+
+    <app-demo>
+      <h2>Page Label</h2>
+      <app-demo-code>
+        <gu-row>
+          <gu-column span=12 flex>
+            <gu-page-label>
+              Article
+
+              <gu-page-label-hover>
+                <a href="#">Back to dashboard</a>
```

Comment:
This is just example code. The point is that you can put any element within the hover state.